ironfish/src/rpc/routes/mining/exportMined.ts

```diff
@@ -57,44 +59,64 @@ router.register<typeof ExportMinedStreamRequestSchema, ExportMinedStreamResponse
     Assert.isNotNull(node.chain.head, 'head')
     Assert.isNotNull(node.chain.latest, 'latest')
 
+    const scanForks = request.data?.forks == null ? false : true
+
     const { start, stop } = BlockchainUtils.getBlockRange(node.chain, {
       start: request.data?.start,
       stop: request.data?.stop,
     })
 
     request.stream({ start, stop, sequence: 0 })
 
+    let promise = Promise.resolve()
+    let waiting = 0
+
     for (let i = start; i <= stop; ++i) {
-      const headers = await node.chain.getHeadersAtSequence(i)
+      promise = promise.then(async () => {
+        const headers = scanForks
+          ? await node.chain.getHeadersAtSequence(i)
+          : [await node.chain.getHeaderAtSequence(i)]
```

Comment:
Again, trivial, but since you're doing the same work either way, would it make sense to do:
```ts
const sequenceHeaders = await node.chain.getHeadersAtSequence(i)
const headers = scanForks ? sequenceHeaders : [sequenceHeaders]
```
?

Reply:
They are actually 2 separate functions, one is plural one is not. The plural one pulls all headers from sequence, the non plural one only pulls the one from the main chain. It's a bit sneaky, but take a look.

Reply:
Ah! I totally missed that `s` in there.